tabular/src/autogluon/tabular/models/text_prediction/text_prediction_v1_model.py

```diff
@@ -195,92 +114,49 @@ def _fit(self, X: pd.DataFrame, y: pd.Series,
+        self.model = TextPredictor(label=self._label_column_name,
+                                   problem_type=self.problem_type,
+                                   path=self.path,
+                                   eval_metric=self.eval_metric,
+                                   verbosity=verbosity)
+        X_train.insert(len(X_train.columns), self._label_column_name, y)
         if X_val is not None:
             X_val.insert(len(X_val.columns), self._label_column_name, y_val)
-        scheduler_options = self.params['hpo_params']['scheduler_options']
-        search_strategy = self.params['hpo_params']['search_strategy']
-        if scheduler_options is None:
-            scheduler_options = dict()
-        if search_strategy.endswith('hyperband'):
-            # Specific defaults for hyperband scheduling
-            scheduler_options['reduction_factor'] = scheduler_options.get(
-                'reduction_factor', 4)
-            scheduler_options['grace_period'] = scheduler_options.get(
-                'grace_period', 10)
-            scheduler_options['max_t'] = scheduler_options.get(
-                'max_t', 50)
-        if X_val is None:
-            # FIXME: v0.1 Update TextPrediction to use all training data in refit_full
-            holdout_frac = default_holdout_frac(len(X), True)
-            X, X_val = random_split_train_val(X, valid_ratio=holdout_frac)
-        train_data = TabularDataset(X,
-                                    column_properties=column_properties,
-                                    label_columns=self._label_column_name)
-        logger.log(15, 'Train Dataset:')
-        logger.log(15, train_data)
-        tuning_data = TabularDataset(X_val,
-                                     column_properties=column_properties,
-                                     label_columns=self._label_column_name)
-        logger.log(15, 'Tuning Dataset:')
-        logger.log(15, tuning_data)
-
-        if time_limit is not None:
-            time_limit = time_limit - (time.time() - time_start)
-
-        # FIXME: Inner error message if no text features is not helpful
-        self.model.train(train_data=train_data,
-                         tuning_data=tuning_data,
-                         resource=resource,
-                         time_limits=time_limit,
-                         search_strategy=search_strategy,
-                         search_options=self.params['hpo_params']['search_options'],
-                         scheduler_options=scheduler_options,
-                         num_trials=self.params['hpo_params']['num_trials'],
-                         console_log=verbosity >= 3,
-                         ignore_warning=verbosity < 3,
-                         verbosity=verbosity-1)
+        assert self.params['hpo_params']['num_trials'] == 1 \
+               or self.params['hpo_params']['num_trials'] is None,\
+            'Currently, you cannot nest the hyperparameter search in text neural network ' \
```

Comment:
Can we clarify this error msg? The msg should also tell user what to do instead.